packages/icons/CHANGELOG.md

```
@@ -2,6 +2,11 @@
 
 Nimbus-icons is an open source icon system built by Nuvemshop / Tiendanube.
 
+## 2026-07-14 `1.25.0` (minor version bump)
+
+#### 🎉 New features
+
+- Adds `chat-unread` icon. ([#478](https://github.com/TiendaNube/nimbus-design-system/pull/478) by [@ignaciomachado](https://github.com/ignaciomachado))
 ## 2026-07-16 `1.26.0`
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)